403.php

```diff
@@ -84,8 +84,9 @@ function listVolumeContent(): array
   <h1>Welcome to SimplyCode!</h1>
   <p>
     This looks like your first visit.
+    <strong>Go to <a href="https://tutorial.dev.muze.nl/"> https://tutorial.dev.muze.nl/ </a> and read "Your first application: TodoMVC" to learn about simplycode.</strong>
```

Comment:
Great stuff! :rocket: 
Two minor points:

1. There seem to be unneeded spaces ` ` around the `https://tutorial.dev.muze.nl/` text.
2.  `simplycode` should be CamelCased: `SimplyCode`

Reply:
fixed